android/src/main/java/com/rnziparchive/RNZipArchivePackage.java

```diff
@@ -18,6 +18,11 @@
   }
 
   @Override
+  public List<Class<? extends JavaScriptModule>> createJSModules() {
+    return Collections.emptyList();
+  }
+
```

Comment:
Yeah, as PR says, it's not needed at 0.47+.

Still thinking is there any possibility to make this works against any versions of RN? That would be awesome since we don't ship a breaking change version.